### DISCUSSION TITLE
How to start Marqo database without using Docker image or Docker compose, using pip install or apt install

### DISCUSSION THREAD

**Discussion by nikhilreddy-pottanig** [milvus-io/milvus#35462] [Q&A and General discussion]
Someone help me to start this vector database

**Comment by nikhilreddy-pottanig** [comment]
Because Cluster I am working on doesn't support more than one image to be provided and for my requirement base image is python:3.11 and on top of that using any commands I can install any vector database. But not able to find any proper way to do it which is working and I need version of milvus DB which as support for sparse vector

For now I am using work around as file based Milvus Db, but concerned about the scalability and Multiple process trying to work with it

**Comment by yhmo** [comment]
Hm...... I suppose you should post the question in the marqo repo:  https://github.com/marqo-ai/marqo
